<ui version="4.0" >
 <class>KwSongdbSongEditWidget_base</class>
 <widget class="QWidget" name="KwSongdbSongEditWidget_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>533</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="labelSongName" >
       <property name="text" >
        <string>Song &amp;Name</string>
       </property>
       <property name="buddy" >
        <cstring>editSongName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="editSongName" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" native="1" name="widget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layoutVersionsToolBar" />
       </item>
       <item>
        <widget class="QListWidget" name="listVersions" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frameVersion" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QFormLayout" name="formLayout_2" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>&amp;Version Name</string>
           </property>
           <property name="buddy" >
            <cstring>editVersionName</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="KLineEdit" name="editVersionName" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>&amp;Writer</string>
           </property>
           <property name="buddy" >
            <cstring>editVersionWriter</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>&amp;Copyright</string>
           </property>
           <property name="buddy" >
            <cstring>editVersionCopyright</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="KLineEdit" name="editVersionWriter" />
         </item>
         <item row="2" column="1" >
          <widget class="KLineEdit" name="editVersionCopyright" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Song &amp;Books</string>
           </property>
           <property name="buddy" >
            <cstring>listSongBooks</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <layout class="QVBoxLayout" name="verticalLayout_7" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="layoutSongBooksToolBar" />
           </item>
           <item>
            <widget class="QListWidget" name="listSongBooks" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupSongBook" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="title" >
          <string>Song Book</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4" >
          <property name="fieldGrowthPolicy" >
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="labelSongBook" >
            <property name="text" >
             <string>Song B&amp;ook</string>
            </property>
            <property name="buddy" >
             <cstring>comboSongBooks</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="KComboBox" name="comboSongBooks" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="labelSongBookNumber" >
            <property name="text" >
             <string>Song N&amp;umber</string>
            </property>
            <property name="buddy" >
             <cstring>spinSongBookNumber</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QSpinBox" name="spinSongBookNumber" >
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>99999</number>
            </property>
            <property name="value" >
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEditSongBooks" >
         <property name="toolTip" >
          <string/>
         </property>
         <property name="text" >
          <string>&amp;Edit Song Books</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupLyrics" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Lyrics</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QTabWidget" name="tabsLyrics" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="tabPosition" >
          <enum>QTabWidget::West</enum>
         </property>
         <property name="currentIndex" >
          <number>1</number>
         </property>
         <widget class="QWidget" name="tabLyricsEditor" >
          <attribute name="title" >
           <string>&amp;Editor</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabLyricsMarkup" >
          <attribute name="title" >
           <string>&amp;Markup</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="layoutLyricsMarkupToolBar" />
           </item>
           <item>
            <widget class="QPlainTextEdit" name="editLyricsMarkup" />
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editSongName</tabstop>
  <tabstop>listVersions</tabstop>
  <tabstop>editVersionName</tabstop>
  <tabstop>editVersionWriter</tabstop>
  <tabstop>editVersionCopyright</tabstop>
  <tabstop>listSongBooks</tabstop>
  <tabstop>comboSongBooks</tabstop>
  <tabstop>spinSongBookNumber</tabstop>
  <tabstop>btnEditSongBooks</tabstop>
  <tabstop>tabsLyrics</tabstop>
  <tabstop>editLyricsMarkup</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
